photon-ml/src/integTest/scala/com/linkedin/photon/ml/function/ObjectiveFunctionIntegTest.scala

```diff
@@ -119,21 +121,17 @@ class ObjectiveFunctionIntegTest extends SparkTestUtils {
    * @return a list of [[LabeledPoint]]
    */
   def generateBenignLocalDataSetBinaryClassification(): List[LabeledPoint] = {
-    val tmp1: List[LabeledPoint] = drawBalancedSampleFromNumericallyBenignDenseFeaturesForBinaryClassifierLocal(
-      ObjectiveFunctionIntegTest.DATA_RANDOM_SEED,
-      ObjectiveFunctionIntegTest.TRAINING_SAMPLES,
-      ObjectiveFunctionIntegTest.PROBLEM_DIMENSION).map({ obj =>
-      assertEquals(obj._2.length, ObjectiveFunctionTest.PROBLEM_DIMENSION, "Samples should have expected lengths")
+    val tmp1 = drawBalancedSampleFromNumericallyBenignDenseFeaturesForBinaryClassifierLocal(
+      DATA_RANDOM_SEED, TRAINING_SAMPLES, PROBLEM_DIMENSION).map({ obj =>
+      assertEquals(obj._2.length, PROBLEM_DIMENSION, "Samples should have expected lengths")
```

Comment:
The original was easier to read. I think the correct Scala style is:
```
val tmp1: List[LabeledPoint] = drawBalancedSampleFromNumericallyBenignDenseFeaturesForBinaryClassifierLocal(
  DATA_RANDOM_SEED,
  TRAINING_SAMPLES,
  PROBLEM_DIMENSION)
  .map({ obj =>
    assertEquals(obj._2.length, ObjectiveFunctionTest.PROBLEM_DIMENSION, "Samples should have expected lengths")
    new LabeledPoint(label = obj._1, obj._2, offset = 0, weight = 1)
  })
  .toList
```
Same goes for similar calls below